app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,19 @@
+class FoodsController < ApplicationController 
+  def index
+    @query = params[:q]
+    key = "X5MqWQCLRSlxUlXnpdm2q51W13hIhOhj5krsN0ep"
+    conn = Faraday.new(url: "https://api.nal.usda.gov") do |faraday|
+      faraday.headers['api_key'] = "X5MqWQCLRSlxUlXnpdm2q51W13hIhOhj5krsN0ep" #ENV['usda_api_key']
+    end
+    
+    response = conn.get("/fdc/v1/foods/search?query=#{@query}&numberOfResultsPerPage=10&api_key=#{key}")
```

Comment:
Would love to see the params set in a block so that this is more readable. Also, not good that you pushed up your API key to github. That's a security vulnerability that can have your permissions from an API revoked. Looks like you set it both as a header and as a param. Generally, you only set that in one of those places depending on what the documentation asks for. Again, this could cause a security vulnerability.